.github/workflows/ci.yml

```diff
@@ -30,7 +30,7 @@ jobs:
       fail-fast: false
 
     steps:
-      - uses: actions/checkout@v3
+      - uses: actions/checkout@v4
```

Comment:
Changelog says they updated from node 16 to node 20, we still use node 16 here.

I think we should update that too then, also node 16 is apperently [EOL](https://nodejs.org/en/blog/announcements/nodejs16-eol)since 11/09/2023

![image](https://github.com/chartjs/Chart.js/assets/39033624/6847ee8e-bee2-44a7-afd5-9cff59d6dffa)

Reply:
Updating the node version sounds good to me.